src/main/java/uk/gov/hmcts/dm/service/StoredDocumentService.java

```diff
@@ -61,24 +63,30 @@ public void saveItemsToBucket(Folder folder, List<MultipartFile> files)  {
         folderRepository.save(folder);
     }
 
-    public List<StoredDocument> saveDocuments(List<MultipartFile> files,
-                                              Classifications classification,
-                                              List<String> roles,
-                                              Map<String, String> metadata)  {
-        return files.stream().map(file -> {
+    public List<StoredDocument> saveItems(UploadDocumentsCommand uploadDocumentsCommand)  {
+        return uploadDocumentsCommand.getFiles().stream().map(file -> {
             StoredDocument document = new StoredDocument();
-            document.setClassification(classification);
-            document.setRoles(roles != null ? roles.stream().collect(Collectors.toSet()) : null);
-            document.setMetadata(metadata);
+            document.setClassification(uploadDocumentsCommand.getClassification());
+            document.setRoles(uploadDocumentsCommand.getRoles() != null
+                ? uploadDocumentsCommand.getRoles().stream().collect(Collectors.toSet()) : null);
+
+            if (toggleConfiguration.getMetadatasearchendpoint()) {
```

Comment:
Should the type be `boolean` instead of `Boolean` so that the getters are formatted in the more standard `isMetadatasearchendpoint`?